src/com/tony/server/response/ImageContentResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.tony.server.response;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+
+public class ImageContentResponse extends Response{
+
+    private final String filePath;
+
+    public byte[] respond(){
+        try {
+            byte[] imageBytes = Files.readAllBytes(Paths.get(filePath));
+            return ResponseBuilder.buildImageResponse(getStatusLine(), getHeaders(), imageBytes);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
I haven't actually tested this. Will I get a 500 response if something goes wrong here? Will I get a 500 if there's any internal server error?